skada/datasets/_office.py

```diff
@@ -470,4 +470,7 @@ def _load_office31(
             data.append(content.astype(loader_spec.dtype))
     files['data'] = np.vstack(data)
```

Comment:
just set X and y here no need to add stuff and pop it

Reply:
We can't. The `files` object comes from the `load_files` function, which is a sklearn function. Thus the keys 'data' and 'target' exists from the beginning.
Thus at some point we would have to remove these keys to not have duplicates with the keys 'X' and 'y'.
Thus the pop methods